File: scripts/ui/ANItex.js
Events.on(ContentInitEvent, () => {
    if(!Vars.headless){
        module.exports["panelClear"] = Core.atlas.drawable("animalia-pane-clear");
        module.exports["sidebarLeft"] = Core.atlas.drawable("animalia-sidebar-left");

        //Icons

        module.exports["frog"] = Core.atlas.drawable("animalia-icon-frog")
    }
});

File: src/animalia/ui/AniTex.java
package animalia.ui;

import arc.scene.style.Drawable;
import static arc.Core.*;

public class AniTex {
    public static Drawable frog;

    public static void load(){
        app.post(() -> {
            frog = atlas.drawable("animalia-icon-frog");
        });
    }
}
